static/js/tools/map/chart.tsx

```diff
@@ -172,6 +172,16 @@ export function Chart(props: ChartProps): JSX.Element {
     };
   }, [props, chartContainerRef]);
 
+  // Triggered only when stat vars or places change and send data to google analytics.
+  useEffect(() => {
+    window &&
+      window.gtag &&
+      window.gtag("event", "tool_chart_plot", {
+        place_dcid: props.placeInfo.enclosingPlace.dcid,
+        stat_var: props.statVar.value.dcid,
+      });
+  }, [props.statVar.value.dcid, props.placeInfo.enclosingPlace.dcid]);
```

Comment:
have you been able to validate that this does not overtrigger?

Reply:
I've tested it in several ways. e.g., only stat var or place is selected, zoom in or out the map, change sources or select per capita. It appears to only trigger once when both stat var and place are selected or any of them change.